src/py_eddy_tracker/dataset/grid.py

```diff
@@ -1315,7 +1315,7 @@ def finalize_kernel(self, kernel, order, half_x_pt, half_y_pt):
 
     def kernel_lanczos(self, lat, wave_length, order=1):
         """Not really operational
-        wave_length in km
+        wavelength in km
```

Comment:
n this case we talked about variable which are called "wave_length", i don't know the best soltuion